authentication/assuming-an-aws-iam-role-for-a-service-account.adoc

```diff
@@ -11,11 +11,12 @@ toc::[]
 
 [role="_abstract"]
 ifdef::openshift-rosa[]
-{product-title} clusters that use the AWS Security Token Service (STS) include a pod identity webhook for use with pods that run in user-defined projects.
+In {product-title} clusters that use the AWS Security Token Service (STS), the OpenShift API server can be enabled to project signed service account tokens which then can be used to assume an AWS Identity and Access Management (IAM) role in a pod. If the assumed IAM role has the required AWS permissions, the pods can authenticate against the AWS API using temporary STS credentials to perform AWS operations.
```

Comment:
Suggestion: "...the OpenShift API server can be enabled to project signed service account tokens that can be used to assume an AWS..."